db/migrate/20200520211047_adjust_chargeback_reports_features.rb

```diff
@@ -0,0 +1,52 @@
+class AdjustChargebackReportsFeatures < ActiveRecord::Migration[5.2]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
+
+  FEATURE_MAPPING = {
+    'chargeback_download_csv'  => 'chargeback_reports_download_csv',
+    'chargeback_download_pdf'  => 'chargeback_reports_download_pdf',
+    'chargeback_download_text' => 'chargeback_reports_download_text',
+    'chargeback_report_only'   => 'chargeback_reports_report_only'
+  }.freeze
+
+  def up
+    return if MiqProductFeature.none?
+
+    say_with_time 'Adjusting chargeback reports features for the non-explorer views' do
+      new_feature = MiqProductFeature.find_or_create_by!(:identifier => 'chargeback_reports_view')
+      MiqRolesFeature.where(:miq_product_feature_id => old_feature_ids).each do |feature|
+        MiqRolesFeature.create!(:miq_product_feature_id => new_feature.id, :miq_user_role_id => feature.miq_user_role_id)
+      end
+
+      # Direct renaming of features
+      FEATURE_MAPPING.each do |from, to|
+        MiqProductFeature.find_by(:identifier => from)&.update!(:identifier => to)
+      end
+    end
+  end
+
+  def down
+    return if MiqProductFeature.none?
+
+    say_with_time 'Adjusting chargeback reports features to explorer views' do
+      admin_feature = MiqProductFeature.find_or_create_by!(:identifier => 'chargeback_reports')
+
+      return if MiqRolesFeature.none?
```

Comment:
This will end up skipping the reverse rename [below](https://github.com/ManageIQ/manageiq-schema/pull/487/files#diff-7c5b5ce01487a3575fbf23a171257674R42)...can you verify this in a spec?

Reply:
addressed [here](https://github.com/ManageIQ/manageiq-schema/pull/487/commits/b7108ad7b1b96e7fbf66a1fa022f8357ab529349#diff-05b72a996a9f2a2185bd92e1d7f3016dR65-R70)

Reply:
I don't see it addressed... basically, even if there are no MiqRolesFeatures, doesn't line 44 still have to happen?  I think this line can actually be removed completely

Reply:
@Fryguy you are right, with that check the code would not have executed line 44, and ideally there will always be MiqRoleFeatures from seeding so i have removed that line. please take a look.